src/Native/Unix/System.Native/pal_networking.h

```diff
@@ -53,16 +53,18 @@ typedef enum
  *
  * NOTE: these values are taken from System.Net.AddressFamily. If you add
  *       new entries, be sure that the values are chosen accordingly.
+ *       Unix specific value have distinct offset to avoid conflict with
+ *       Windows values.
  */
 typedef enum
 {
     AddressFamily_AF_UNSPEC = 0,   // System.Net.AddressFamily.Unspecified
     AddressFamily_AF_UNIX = 1,     // System.Net.AddressFamily.Unix
     AddressFamily_AF_INET = 2,     // System.Net.AddressFamily.InterNetwork
     AddressFamily_AF_INET6 = 23,   // System.Net.AddressFamily.InterNetworkV6
-    AddressFamily_AF_NETLINK = 30, // System.Net.AddressFamily.Netlink
-    AddressFamily_AF_PACKET = 31,  // System.Net.AddressFamily.Packet
-    AddressFamily_AF_CAN = 32,     // System.Net.AddressFamily.ControllerAreaNetwork
+    AddressFamily_AF_NETLINK = 65536, // System.Net.AddressFamily.Netlink
+    AddressFamily_AF_PACKET = 65537,  // System.Net.AddressFamily.Packet
+    AddressFamily_AF_CAN = 65538,     // System.Net.AddressFamily.ControllerAreaNetwork
```

Comment:
Am I remembering correctly that we were going to back out shipping one of these because there was no way to use it? If so, when is that going to happen?

Reply:
I thought we discussed this whole feature over email.  We were either going to pull it all out.  Or fix it.  We chose the later. This PR represents the fix of the feature.

But I remember this comment from @krwq on the email:

>+1 for option 1. Also might be a good idea to try to make it so that linux_value = enum_value << 16
So we can also quickly cast options on linux but consecutive numbers are fine too with me since this is 1 time operation anyway.

@wfurt Does this PR incorporate that feedback as well?

Reply:
> We were either going to pull it all out. Or fix it. We chose the later. This PR represents the fix of the feature

No, I'm talking about this:
https://github.com/dotnet/corefx/pull/39785#discussion_r307519430

Reply:
I updated code to reflect that @stephentoub. The current change should be minimal reasonable set. With the change we should have no problem moving this forward in future releases.

As far as the specific value. It seems like the general agreement was that using platform values is not great and that is causing  current issues. That gets also more complicated when more OSes support same protocol. So I plan to keep consecutive values independent of actual platform values.